skipchain/struct.go

```diff
@@ -629,12 +629,12 @@ func (db *SkipBlockDB) GetByID(sbID SkipBlockID) *SkipBlock {
 func (db *SkipBlockDB) StoreBlocks(blocks []*SkipBlock) ([]SkipBlockID, error) {
 	var result []SkipBlockID
 	err := db.Update(func(tx *bolt.Tx) error {
-		fl := blocks[len(blocks)-1].ForwardLink
-		if len(fl) > 0 {
-			if db.GetByID(fl[len(fl)-1].To) == nil {
-				return errors.New("can't have last block pointing into empty space")
-			}
-		}
+		// fl := blocks[len(blocks)-1].ForwardLink
```

Comment:
remove please

Reply:
I was hesitating to remove that one - but with the new catch-up mechanism, it can make sense to store blocks that have links to non-existing blocks. So anybody speak up if he thinks it is a bad idea to store blocks that have forward-links to other unknown blocks.